### DISCUSSION TITLE
[Docs] Where can I find the live code for Magma Docs?

### DISCUSSION THREAD

**Discussion by ShubhamTatvamasi** [magma/magma#5324] [Q&A] [answered]
I was just comparing these two files and I found a difference in the `helm template orc8r` command.

https://github.com/magma/magma/blob/master/docs/readmes/orc8r/development/minikube_deployment.md#generate-secrets

https://docs.magmacore.org/docs/next/orc8r/development/minikube_deployment#generate-secrets

**Comment by karthiksubraveti** [comment, marked as answer]
docs/readmes is the location from which magma docs is generated. If you make documentation changes under readme, it should be reflected in the docusaurus.

There is also a script to create a local copy of the Docusaurus for validation https://github.com/magma/magma/blob/1e7575015b2929e5df2f6bb6adaee89e0155c8b4/docs/docusaurus/create_docusaurus_website.sh

**Reply by ShubhamTatvamasi** [reply]
Then why I see `helm template orc8r charts/secrets` on [github](https://github.com/magma/magma/blob/master/docs/readmes/orc8r/development/minikube_deployment.md#generate-secrets).
And `helm template orc8r-secrets charts/secrets` on [website](https://docs.magmacore.org/docs/next/orc8r/development/minikube_deployment#generate-secrets)?

**Reply by karthiksubraveti** [reply]
It was updated recently - https://github.com/magma/magma/pull/5233/files and docs website wasn't probably regenerated

**Reply by hcgatewood** [reply]
For additional context: here's where we generate the GitHub pages site https://github.com/magma/magma/blob/11f81902d3d8ebccf76d844adabdd94f875b3926/.circleci/config.yml#L947-L972

**Reply by ShubhamTatvamasi** [reply]
Is there is some bug that's why the website is not regenerating properly?

**Reply by hcgatewood** [reply]
There are two different deployments of the Docusaurus docs. The "original" one is deployed via GitHub Pages, and is the job I referenced above. It is functioning properly: https://magma.github.io/magma/docs/next/orc8r/development/minikube_deployment

The link you provided is a Netlify-based deployment handled by other parts of the community. @tmdzk has more context on this, and pointers to who can help un-break
